src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -1333,5 +1333,11 @@ public Version getVersion()
          */
         public AtomicInteger numEndpointsNoMessageTransportAfterDelay =
             new AtomicInteger();
+
+        /**
+         * The total number of times the dominant speaker in any conference
+         * changed.
+         */
+        public AtomicInteger totalDominantSpeakerChanges = new AtomicInteger();
```

Comment:
Would this be better as a `LongAdder` rather than an `AtomicLong`?

Reply:
Because of contention? Dominant speaker changes don't happen very often.

Reply:
Not a big deal either way, I guess, I just thought `LongAdder` was supposed to be more efficient when you don't need test-and-set semantics.